### PR TITLE
clarifications

### PR DESCRIPTION
**Pull request by Soheab**
adding the imports under root didn't work anymore.

and added the other text to let people know its possible

### CODE REVIEW

File: autotheme.css [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+  3. Enjoy some fun stuff :P
+*/
 
 :root {
   --accent: #7a78bd;
   --dark-highlight: #181818;
   --dark-primary: #212121;
   --dark-secondary: #303030;
-  --discord-logo: 1;
+  --discord-logo: 1; /* change this to 0 to remove the discord logo+ the stripe */
```
Comment (AlexFlipnote) [inline autotheme.css]: - Add some spacing with the +
- Capital C

File: autotheme.css [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
   --light-dark-text: #2e3136;
   --light-highlight: #9e9e9e;
   --light-primary: #ddd;
   --light-secondary: #ecf0f1;
   --notif: #7a78bd;
-  --theme-logo: 1;
+  --theme-logo: 1; /* change this to 0 to remove the theme text+version*/
```
Comment (AlexFlipnote) [inline autotheme.css]: - Add some spacing on both sides to the +
- Some space at the end of the comment
- Capital C

File: autotheme.css (hunk 1 above)
Comment (Soheab) [reply autotheme.css]: done

File: autotheme.css (hunk 2 above)
Comment (Soheab) [reply autotheme.css]: done

**Pull request merged**